docs/providers/aws/guide/functions.md

```diff
@@ -253,6 +253,29 @@ functions:
       TABLE_NAME: tableName2
 ```
 
+## KMS Key
+
+AWS Lambda uses [AWS Key Management Service (KMS)](https://aws.amazon.com/kms/) to encrypt your environment variables at rest. You can override the default KMS key used by the Lambda function at the provider or function level:
+
+```yml
+# serverless.yml
+service: service-name
+provider:
+  name: aws
+  kmsKeyArn: [your KMS key ARN]
+
+functions:
+  hello: # this function will INHERIT the service level environment config above
+    handler: handler.hello
+  users: # this function will OVERWRITE the service level environment config above
+    handler: handler.users
+    kmsKeyArn: [your KMS key ARN]
```

Comment:
Addressed in https://github.com/serverless/serverless/pull/2998/commits/8fe133283ef6aacf1c45cb4a678f9bccb1c362b3